src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/info-card/info-card-popover.scss

```diff
@@ -17,7 +17,7 @@
       }
 
       span.health-warn-description {
-        color: vv.$health-color-warning-wcag-aa-regular-text !important;
+        color: darken(vv.$health-color-warning, 9) !important;
```

Comment:
We shouldn't define new colors in per-component CSS files. Everything should be defined in the top-level stylesheet.

Additionally, while this is an interesting approach, it might break downstream/vendor brandings. What most CSS frameworks ([Bootstrap 5](https://getbootstrap.com/docs/5.0/utilities/colors/), [Material 3](https://m3.material.io/styles/color/the-color-system/key-colors-tones), [Patternfly 4](https://www.patternfly.org/v4/guidelines/colors/#color-palette))  do instead is using color variants based on different tones: `<colorname>-XXX`.

Reply:
Thanks  @epuertat this is definitely an 800 or 900 shade